src/components/sidebar/routePathView/NewRoutePathView.tsx

```diff
@@ -75,27 +67,46 @@ class NewRoutePathView extends React.Component<INewRoutePathViewProps>{
         this.props.networkStore!.setIsLinksVisible(true);
 
         this.props.routePathStore!.setIsCreating(true);
+
+        // TODO: this should be action call
```

Comment:
I also find this lifecycle a bit difficult to understand, would it be possible to move some logic in this to other functions as well?

Reply:
Already fixed this